### DISCUSSION TITLE
The time is completely cut off on the right side. Please help me fix this.

### DISCUSSION THREAD

**Discussion by MrCherub** [catppuccin/tmux#359] [Help] [answered]
![Image 10-5-24 at 12 40 AM](https://github.com/user-attachments/assets/459deddf-d2e2-43f7-ae66-295ee9ac16dc)

**Comment by kjnsn** [comment]
Okay so quite a few comments here:

1. In future, please copy-paste your config as text. Having to zoom into a screenshot is really difficult, and innaccessible. 
2. On line 17 you have misspelled `catppuccin`. There are two "p"s
3. Please don't set defaults again. You can remove the `default fill` options.
4. The status style isn't being applied because you have two `@@`
5. Why are you setting `"#{E:@catppuccin_status_application} #S"`? This is already set on the right.
6. Don't use `-F` when using those status line modules. That's why you're seeing `tmux` instead of `nvim`. This is mentioned in the documentation.
7. Use `tmux-256color` instead of `screen-256color`. Screen is absolutely ancient.
8. You are not running catppuccin where the comment tells you to on line 38.

Also please don't nearly post a copy of your DNA onto the internet....

**Reply by MrCherub** [reply]
LOL thank you. I'm new to all of this.

**Reply by MrCherub** [reply]
> Okay so quite a few comments here:
> 
> 1. In future, please copy-paste your config as text. Having to zoom into a screenshot is really difficult, and innaccessible.
> 2. On line 17 you have misspelled `catppuccin`. There are two "p"s
> 3. Please don't set defaults again. You can remove the `default fill` options.
> 4. The status style isn't being applied because you have two `@@`
> 5. Why are you setting `"#{E:@catppuccin_status_application} #S"`? This is already set on the right.
> 6. Don't use `-F` when using those status line modules. That's why you're seeing `tmux` instead of `nvim`. This is mentioned in the documentation.
> 7. Use `tmux-256color` instead of `screen-256color`. Screen is absolutely ancient.
> 8. You are not running catppuccin where the comment tells you to on line 38.
> 
> Also please don't nearly post a copy of your DNA onto the internet....

It's much worse now. What am I missing? 
<img width="1920" alt="Screenshot 2024-10-05 at 3 40 18 AM" src="https://github.com/user-attachments/assets/3d5e1541-f495-4192-acc2-dce0a859fbc4">
set -g default-terminal "tmux-256color"

set -g prefix C-a

unbind C-b

bind-key C-a send-prefix
unbind %
bind Space split-window -v


set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-sensible'
set -g @plugin 'christoomey/vim-tmux-navigator'
set -g @plugin 'catppuccin/tmux'
set -g @catppuccin_flavour 'mocha'
set -g @catppuccin_window_status_style "slanted"
set -g @catppuccin_window_number_position "right"

set -g @catppuccin_window_current_text "#{pane_current_path}"

set -g @catppuccin_status_left_separator  " "
set -g @catppuccin_status_right_separator " "
set -g @catppuccin_status_fill "all"
set -g @catppuccin_status_connect_separator "yes"


set -g status-left ""
set -g status-right "#{E:@catppuccin_status_application}#{E:@catppuccin_status_session}#{E:@catppuccin_status_date_time}"
run ~/.config/tmux/plugins/catppuccin/tmux/catppuccin.tmux


Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
run '~/.tmux/plugins/tpm/tpm'

**Reply by kjnsn** [reply]
I disagree that this is "much worse". What are you expectations?

The status line length is controlled with the option `status-right-length` as documented by tmux: https://man7.org/linux/man-pages/man1/tmux.1.html#OPTIONS

Again, please _only_ take a screenshot of the terminal. If you post a picture of your DNA again, I will remove it: https://support.apple.com/en-us/102646

There is a bug with the slanted style and having the number on the right (#353). You are welcome to submit a PR fixing it

You also didn't load catppuccin in the correct place, but everything seems to work anyway so...

**Reply by RayGuo-ergou** [reply, marked as answer]
It's likely caused by this: https://github.com/catppuccin/tmux/commit/a1ec384c11c5cbab755f77340c4e843945474afd#diff-92148f9086d6ca4a819d45bf69b4a16b22a0e67381f9f0a498f613d658d5eb82L19

You can try to add it to your config.

**Reply by kjnsn** [reply]
Can you elaborate? Add what to your config? What was caused by what?

**Reply by RayGuo-ergou** [reply]
Isn't the url to a specific line of commits diff self-explanatory enough?

So because `status-{left,right}-length` is removed recently that caused long status bar is not showing everything.

**Reply by kjnsn** [reply]
It's not particularly. 

The screenshot was made before I removed the status line length options.

**Comment by xdannyrobertsx** [comment]
adding the following to my config fixed this for me:
```
set -g status-left-length "100"
set -g status-right-length "100"
```

the above were removed from the defaults in the commit linked above - this is a welcomed changed since it gives the user more control, but could throw newer users of this module off if they started using it right before the migration (aka me lol)
